config_src/nuopc_driver/mom_surface_forcing_nuopc.F90

```diff
@@ -1094,7 +1094,7 @@ subroutine surface_forcing_init(Time, G, US, param_file, diag, CS, restore_salt,
     call get_param(param_file, mdl, "USE_NET_FW_ADJUSTMENT_SIGN_BUG", &
                  CS%use_net_FW_adjustment_sign_bug, &
                    "If true, use the wrong sign for the adjustment to "//&
-                   "the net fresh-water.", default=.false.)
+                   "the net fresh-water.", default=.true.)
```

Comment:
Same here. The default should be False.

Reply:
Addressed in 38710730114452d0111214c72f274a0f8628f451